cfme/tests/cloud_infra_common/test_power_control_manual.py

```diff
@@ -3,9 +3,9 @@
 
 from cfme import test_requirements
 
+pytestmark = [test_requirements.power, pytest.mark.manual]
```

Comment:
Moving `pytest.mark.manual` to pytestmark is a good idea, but I think you should keep `test_requirements.power` must be with individual tests, it will be easier when you move the test case out of manual. Although, you can ignore this if the file that you're moving the test case to has test_requirements in the pytestmark.

Reply:
I agree with you but as you said, the files that I am moving this test case to has  test_requirements in the pytestmark.